lib/lens.ex

```diff
@@ -3,7 +3,7 @@ defmodule Lens do
 
   @opaque t :: (:get, any, function -> list(any)) | (:get_and_update, any, function -> {list(any), any})
 
-  @doc """
+  @doc ~S"""
```

Comment:
Should have made 2 commits here. A commit with "and" in the title is a red flag.